datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -191,3 +191,24 @@ query P
 SELECT DATE_TRUNC('second', '2022-08-03 14:38:50Z');
 ----
 2022-08-03T14:38:50
+
+# Test that interval can add a timestamp
```

Comment:
Thank you!

I also suppose it's worth adding tests for `interval + timestamp` case, as it was added in scalar.rs

Reply:
I see tests here for `timestamp + interval` and `interval + timestamp` (e.g. `SELECT INTERVAL '8' DAY + timestamp '2013-07-01 12:00:00';`)

I don't see tests for `date + timestamp` and `timestamp + date` -- maybe that is what you meant?

Reply:
I meant last four tests, added in 2nd commit